stable/datacube-dashboard/values.yaml

```diff
@@ -23,10 +23,10 @@ resources:
     memory: 1024Mi
 dockerArgs: [ "gunicorn", "-b", "0.0.0.0:8080", "-w", "4", "--timeout", "60", "cubedash:app" ]
 annotations:
-  iam.amazonaws.com/role: user.dev-eks-datacube
+  iam.amazonaws.com/role: dev-eks-datacube-cubedash
```

Comment:
No need to create a new one you can use existing role for wms

Reply:
Existing role does not have sufficient permission to use `EKS`/`S3`/`RDS`/`CloudWatchEvents` resource